cmd/ncproxy/ncproxy.go

```diff
@@ -151,77 +102,83 @@ func (s *grpcService) ModifyNIC(ctx context.Context, req *ncproxygrpc.ModifyNICR
 		trace.StringAttribute("endpointName", req.EndpointName),
 		trace.StringAttribute("nicID", req.NicID))
 
-	log.G(ctx).WithField("iov settings", req.IovPolicySettings).Info("ModifyNIC iov settings")
```

Comment:
Is logging these settings lost, or are they logged elsewhere?

Reply:
hmm I just removed that log entirely, but I can add it back in later